docs/docs/governance/unlock-dao-tokens.mdx

```diff
@@ -119,3 +119,91 @@ At that point, the collected fees can be "swapped and burned" by calling the `sw
+
+The **UP token** (Unlock Protocol Token) is the native governance token of the Unlock Protocol.
+
+## What is the Unlock Protocol UP token used for?
+
+The UP token serves several key functions:
+
+1. **Incentives**: The UP token is used to reward developers and creators who build on or integrate the Unlock Protocol. For example, developers who deploy smart contracts on the platform may earn UP tokens as rewards.
+2. **Governance**: UP token holders can participate in the governance of the protocol by voting on proposals or protocol changes.
+
+## How do you get the Unlock Protocol UP token?
+
+There are multiple ways for individuals to get the UP token.
+
+- **Swap UDT for UP** — Holders of UDT can swap UDT for UP
+- **Purchase UP directly** — The Unlock Protocol UP token may be acquired on decentralized exchanges such as Uniswap
+- **Receive UP as grants from the Unlock DAO** — The Unlock DAO has a grants program, where builders can request grants of UP for projects built using Unlock Protocol
+- **Protocol rewards** — Transactions that use Unlock Protocol that contain a financial component can include a referrer fee, paid in the Unlock Protocol UP token
+
+## What blockchain network is the Unlock Protocol UP token issued on?
+
+The UP token is on the [Base](https://base.org) blockchain network.
+
+## What is the contract address of the Unlock Protocol UP token?
+
+The contract for the UP token is [https://basescan.org/token/0xac27fa800955849d6d17cc8952ba9dd6eaa66187#code](https://basescan.org/token/0xac27fa800955849d6d17cc8952ba9dd6eaa66187#code)
+
+## What is the supply of the Unlock Protocol UP token?
+
+The UP token on Base has a supply of 1,000,000,000 (one billion) tokens.
+
+## What are the tokenomics of the UP token?
+
+Holdings of the UP token are distributed across a number of different groups in the Unlock Protocol ecosystem, including:
+
+- Unlock Labs
+- Community
+- DAO
+- Liquidity Pools
+- Airdrops & Campaigns
+
+TO DO: PUT PIE CHART HERE
+
+## Will the supply of UP change over time?
+
+The maximum supply of the Unlock Protocol UP token is fixed at the upper bound of 1,000,000,000 tokens. Over time, the Unlock Protocol does collect a protocol fee on financial transactions that use the protocol. These fees are can be used as component of a buyback-and-burn mechanism, that may reduce the total supply of UP tokens over time.
+
+## Are there liquidity pools for the Unlock Protocol UP token?
+
+There is a Uniswap V3 liquidity pool for Base ETH/UP pairs at [https://app.uniswap.org/explore/pools/base/0x9EF81F4E2F2f15Ff1c0C3f8c9ECc636580025242](https://app.uniswap.org/explore/pools/base/0x9EF81F4E2F2f15Ff1c0C3f8c9ECc636580025242)
+
+## What is the difference between the Unlock Protocol UP token (UP) and the Unlock DAO Token (UDT)
+
+UP token transactions are faster and have significantly lower transaction (gas) fees than UDT transactions. This is as a result of the UP token being on Base, rather than on Ethereum mainnet.
```

Comment:
I don't think that's correct and that does not really convey the difference between UDT and UP. (Tehcnically you can swap UDT on Base "as fast" as UP on Base).

I'd say the differences are:
- "Canonical" UP has been deployed on Base and benefits from faster transactions while still allowing governance. 
- UP has 1,000,000,000 supply while UDT had a supply for 1,000,000 and UDT/UP has a fixed "exchange" rate for 1 for 1000.
Outside of this the tokens are the same and since you can always exchange one for the other, they are the same from a speculation perspective.

Reply:
copy updated to above bullets